sdk/postgresql/arm-postgresql-flexible/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
+    
+## 8.0.0 (2023-04-10)
+    
+**Features**
 
-## 7.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Enum KnownArmServerKeyType has a new value SystemManaged
 
-### Other Changes
+**Breaking Changes**
 
+  - Enum KnownArmServerKeyType no longer has value SystemAssigned
```

Comment:
@MaryGao are these non-breaking changes? If they are, I should change these to Features.

Reply:
no, they are real breaking change caused by this PR https://github.com/Azure/azure-rest-api-specs/pull/23294